pkg/controller/dataimportcron-controller_test.go

```diff
@@ -340,6 +343,12 @@ var _ = Describe("All DataImportCron Tests", func() {
 			Expect(*dv.Spec.Source.Registry.URL).To(Equal(testRegistryURL + "@" + testDigest))
 			Expect(dv.Annotations[cc.AnnImmediateBinding]).To(Equal("true"))
 
+			if cron.Spec.Schedule == emptySchedule {
+				exists, err := reconciler.cronJobExistsAndUpdated(context.TODO(), cron)
```

Comment:
I think for the unit test it makes sense to write the GET request ourselves, in case `cronJobExistsAndUpdated` ever has an issue. Just adds to the test's integrity IMO